src/main/java/mate/academy/springintro/dto/user/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,21 @@
+package mate.academy.springintro.dto.user;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import lombok.Data;
+import org.hibernate.validator.constraints.Length;
+
+@Data
+public class UserRegistrationRequestDto {
+    @NotBlank
+    @Email
+    private String email;
```

Comment:
there are should be more fields 
Let's look at task description

{
  "email": "john.doe@example.com",
  "password": "securePassword123",
  "repeatPassword": "securePassword123",
  "firstName": "John",
  "lastName": "Doe",
  "shippingAddress": "123 Main St, City, Country"
}